### ISSUE TITLE
HSPC deployment configuration?

### ISSUE THREAD

**Issue by preston** [HL7-DaVinci/PDex-Patient-Import-App#4]
In my notes I have that this app needs to be set up for HSPC deployment. If that's still the case,  what values should be used for production?

**Comment by yuflyud** [comment]
There is nothing really specific to HSPC there, just OAuth2 and FHIR R4. We haven't tested it with other providers though. Try to look at application.properties, you can specify there your OAuth2 config and some other settings. We will update a README with more instructions about configuration this week I hope.

**Comment by preston** [comment]
@VietDNguyen Is this project also one we need to run?

**Comment by VietDNguyen** [comment]
Yes. It’s being built by HealthLX, a Da Vinci member, as one of the Da Vinci reference implementations. I copied Charlie Provenzano, CTO for HealthLX.
V

Viet Nguyen, MD

**Comment by dhasilin** [comment]
Working on that.

**Comment by preston** [comment]
@Kravs Ok -- just reach out via the "davincirefimp" list when it gets to that state.

**Comment by dhasilin** [comment]
@preston I guess 2 alerts repos are ready to be deployed.
I have sent you an email 2 days ago with details :)
I'm just not sure I have access to davincirefimp list.
